bus-mapping/src/util.rs

```diff
@@ -0,0 +1,17 @@
+//! Common utility traits and functions.
+
+use num::BigUint;
+use pasta_curves::arithmetic::FieldExt;
+
+/// Serializes a field element
+pub fn serialize_field_ext<S, F>(
+    v: &F,
+    serializer: S,
+) -> Result<S::Ok, S::Error>
+where
+    S: serde::Serializer,
+    F: FieldExt,
+{
+    serializer
+        .serialize_str(&BigUint::from_bytes_le(&v.to_bytes()).to_str_radix(16))
+}
```

Comment:
I belive you don't need to go through all this to generate a FieldElement as `FieldExt`.

Using `EvmWord::{from,to}_bytes_{le,be}()` you should be able to avoid needing the `BigUint` dep as well as maybe even the serde method (Not sure on the last statement).